requirements.txt

```diff
@@ -1 +1,3 @@
-flask==3.0.3
\ No newline at end of file
+flask==3.0.3
+git
+lib-version @ git+https://github.com/REMLA24-Team-5/lib-version
```

Comment:
2.518 ERROR: lib-version@ git+https://github.com/REMLA24-Team-5/lib-version from git+https://github.com/REMLA24-Team-5/lib-version (from -r requirements.txt (line 3)) does not appear to be a Python project: neither 'setup.py' nor 'pyproject.toml' found

It seems like we need to update lib-version otherwise this does not work

Reply:
yeah, this is because the main branch is still empty for lib-version, as the PRs have not been merged yet

Reply:
Yeah so I think if we merge them this one can be merged aswell